src/h2load.cc

```diff
@@ -2539,10 +2687,17 @@ int main(int argc, char **argv) {
   double rps = 0;
   int64_t bps = 0;
   if (duration.count() > 0) {
-    auto secd = std::chrono::duration_cast<
-        std::chrono::duration<double, std::chrono::seconds::period>>(duration);
-    rps = stats.req_success / secd.count();
-    bps = stats.bytes_total / secd.count();
+    if (config.warm_up_time > 0) {
+      // we only want to consider the rate-period if warm-up is given
+      rps = stats.req_success / config.rate_period;
```

Comment:
As mentioned elsewhere, I think this will be clearer when we have our own `main-phase-time` argument (or whatever we call it).